### PR TITLE
Show back and forward button on WebView only if it is possible to moe back or forward

### PR DESCRIPTION
**Pull request by mukhole**
We use the properties on Windows.Xaml.Control.WebView, canGoForward and canGoBack to determine when to show these buttons.

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1546)

### CODE REVIEW

File: Frameworks/SafariServices/SFSafariWebViewController.mm [hunk 1]
```diff
@@ -177,24 +177,46 @@ - (void)loadView {
...
+
+    if ([_webView canGoBack] == YES) {
+        // <-
+        UIBarButtonItem* backButton =
+            [[UIBarButtonItem alloc] initWithImage:backArrow style:UIBarButtonItemStylePlain target:_webView action:@selector(goBack)];
+        [toolbarItemsArray addObject:backButton];
+    } else {
+        // This is to ensure that the other buttons are rendered at the correct position.
+        [toolbarItemsArray addObject:space];
+    }
```
Comment (jaredhms) [inline Frameworks/SafariServices/SFSafariWebViewController.mm]: This only works when we load the view; what happens after we navigate in the WebView?  Don't we want to update our buttons as we navigate around?

Comment (mukhole) [inline Frameworks/SafariServices/SFSafariWebViewController.mm]: Yes, I misunderstood the way webview loads.
Will update and send another PR.

**Comment by pradipd** [comment]
:shipit:
